puzzles/models.py

```diff
@@ -62,20 +62,23 @@ class Puzzle(models.Model):
 
     # Deprecated. Use guesses instead (see answers/models.py) which points to an Answer
     # object.
-    answer = models.CharField(max_length=128)
+    # TODO: remove this
+    answer = models.CharField(max_length=128, blank=True)
 
-    tags = models.ManyToManyField(PuzzleTag, related_name="puzzles")
+    tags = models.ManyToManyField(PuzzleTag, related_name="puzzles", blank=True)
 
-    metas = models.ManyToManyField("self", symmetrical=False, related_name="feeders")
+    metas = models.ManyToManyField(
+        "self", symmetrical=False, related_name="feeders", blank=True
+    )
 
     is_meta = models.BooleanField(default=False)
 
     active_users = models.ManyToManyField(
```

Comment:
not related to your change, but what was active_users for again?

Reply:
We're not actually using it, lol. I think the idea was to keep track of who's working on a puzzle at a time. We should probably go through and do some cleanup of the stuff we never ended up using or that we deprecated.

Reply:
lol I already forgot I added this feature back in #80...
yeah, doesn't seem like this is used anymore, we can probably just get rid of it

Reply:
Added #599 to clean up `active_users`.